src/main/resources/templates/index.html

```diff
@@ -17,6 +17,26 @@ <h1>Welcome to Neighborhood Assurance!</h1>
 		<input type="text" name="state" placeholder="Test: Search for OH"/>
 		<button name="submit" type="submit" value="0">Submit</button>
 	</form>
+	<br/>
+	<br/>
+Search is not working yet, please use this button

```

Comment:
What is needed to make it work?  That would be a good suggestion for the group. 

Indicating something is not working is like the old "under construction" gifs that were common on webpages in the late 1990s... if it is not ready, don't show it.